policies/trademark-policy.md

```diff
@@ -3,94 +3,126 @@ status: draft
 ---
 # Trademark Policy
 
-This document is a statement regarding the use and licensing of trademarks owned by the CF and used by CF projects. For purposes of this document, references to trademarks or marks include all trade and service marks and logos owned by the Commonhaus Foundation legal entity.
+This document provides guidelines for the use of the Commonhaus Foundation's (CF) trademarks, ensuring they are used in a manner that promotes our open-source projects while maintaining their integrity and value.
 
-## Trademark Definitions and Ownership
+Our trademarks, including the Commonhaus Foundation name, logo, and any project names, represent the quality and reliability our community has come to expect.
+If you have any questions or need further clarification, please [contact us](#contact-and-further-information).
 
-The Commonhaus Foundation owns all CF trademarks, both registered and unregistered. A comprehensive list of registered trademarks, pending registrations, and trademarks in use can be found at <https://trademark-list.website>.
+## Covered Trademarks
 
-### Ownership and Specific Project Trademarks
+This policy applies to all trademarks and service marks owned by the Commonhaus Foundation (collectively referred to as the “Marks”), including but not limited to:
 
-While all trademarks are owned by the Commonhaus Foundation, each CF project may have its own (unique) set of trademarks. These trademarks, along with any specific usage guidelines, will be documented in the project's `TRADEMARK.md` file.
+- **Word Marks:** "Commonhaus Foundation" and any names of projects under our umbrella.
+- **Logos:** The official Commonhaus Foundation logo and logos of individual projects.
+- **Trade Dress:** The distinctive visual appearance of our website, product packaging, or promotional materials.
 
-In case of any inconsistency between the rules in a project's `TRADEMARK.md` file and this Trademark Policy, the project's specific rules will prevail.
+Some Marks may not be registered, but registration does not equal ownership of trademarks.
```

Comment:
Is this saying even if a Mark isn't registered, the CF can own the trademark?

I was a little confused by the wording

Reply:
https://grr.com/publications/who-should-be-the-owner-of-a-registered-trademark-an-individual-or-a-business-entity/

I can register a trademark, but that doesn't mean *I* own it. Also, you can have rights associated with a trade or service mark without registering it.

Language comes from Model Guidelines:

> This Policy encompasses all trademarks and service marks, whether Word Marks, Logos or Trade Dress, which are collectively referred to as the “Marks.” Some Marks may not be registered, but registration does not equal ownership of trademarks. This Policy covers our Marks whether they are registered or not. 

Unregistered trademark: https://en.wikipedia.org/wiki/Unregistered_trademark

Reply:
Thanks for clarifying